src/framework/core/js/DataBinding.js

```diff
@@ -417,8 +439,8 @@ var fluid_2_0 = fluid_2_0 || {};
         if (!transId) {
             fluid.fail("Cannot get transaction record without transaction id");
         }
-        var transRec = instantiator.modelTransactions[transId];
-        if (!transRec && !instantiator.free) {
+        var transRec = instantiator && instantiator.modelTransactions[transId];
+        if (!transRec && instantiator) {
```

Comment:
We check <code>instantiator</code> twice here. It looks like this could be rewritten as:

    if (instantiator && !instantiator.modelTransactions[transId]) { ... }